README.md

```diff
@@ -19,7 +19,10 @@ We want to create website with Django, that will be useful to as many as three d
   - place to submit feedback about meetings,
   - assignments given on meetings.
 - For Genshin Impact players:
-  - TBD
+  - wish simulator with character collection book,
```

Comment:
Add more information for normal (not addicted to genshin) people. IDK what are those "wishes" :(

Reply:
wish is like lootbox, should I add this information to text??

Reply:
yes